sdk/healthbot/Microsoft.Azure.Management.Healthbot/src/Microsoft.Azure.Management.Healthbot.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
```

Comment:
Please also include assemblyinfo.cs for the SDK project. Please refer other SDK for examples.

Reply:
Added.